handlers.go

```diff
@@ -401,3 +402,36 @@ func HTTPMethodOverrideHandler(h http.Handler) http.Handler {
 		h.ServeHTTP(w, r)
 	})
 }
+
+// CacheControlHandler wraps and returns a http.Handler which checks the url
+// extension and adds a Cache-Control header if valid. Headers will only be
+// added to GET requests. The primary use case would be to wrap a static file
+// server.
+//
+// The maximum age is based on the timings defined in h5bp/server-configs-nginx
+// https://github.com/h5bp/server-configs-nginx/blob/master/h5bp/location/expires.conf
+func CacheControlHandler(h http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		if r.Method == "GET" {
+			var age time.Duration
+			ext := filepath.Ext(r.URL.String())
+
+			switch ext {
```

Comment:
Where do these extensions come from / how are they decided? If it's for wrapping a static file server, should we  not just indicate that everything will be cached?